cmd/serve.go

```diff
@@ -8,10 +8,11 @@ import (
 	"os"
 	"os/exec"
 	"os/signal"
+	"path/filepath"
 	"time"
 
 	gocmd "github.com/go-cmd/cmd"
-	packr "github.com/gobuffalo/packr/v2"
+	"github.com/gobuffalo/packr/v2"
```

Comment:
@fadeev this is reverted back again. it seems that I was using an old version of probably `goimports` in my laptop.